SIDFactoryII/source/utils/global.cpp

```diff
@@ -0,0 +1,57 @@
+#include "global.h"
+#include "configfile.h"
+#include "foundation/base/assert.h"
+#include "foundation/platform/platform_factory.h"
+#include <string>
+#include <vector>
+
+using namespace Foundation;
+
+namespace Utility
+{
+
+	Global& Global::instance()
+	{
+		static Global instance;
```

Comment:
This is a funky construct. It works, because the static Global variable is not instantiated until the first time Global::instance() is called.

Reply:
Funky as in cool, or let's do it another way? ;)